torch/distributed/_composable/fsdp/_fsdp_param_group.py

```diff
@@ -600,6 +600,15 @@ def _validate_cpu_offload_params(self):
                 "Found following parameters on non-CPU device: "
                 f"{[(fsdp_param._param_fqn, fsdp_param.sharded_param.device) for fsdp_param in fsdp_params_not_on_cpu]}\n"
             )
+        backend = dist.get_backend()
+        backend_capability = dist.Backend.backend_capability.get(backend)
+        if (
+            backend_capability and "cpu" not in backend_capability
+        ) or "cpu" not in backend:
```

Comment:
Curious: what does `"cpu" not in backend` mean?
`backend` should be enums like `NCCL`, `GLOO`, etc.